src/cdk/table/sticky-styler.ts

```diff
@@ -50,7 +51,14 @@ export class StickyStyler {
                */
               private _coalescedStyleScheduler?: _CoalescedStyleScheduler,
               private _isBrowser = true,
-              private readonly _needsPositionStickyOnElement = true) { }
+              private readonly _needsPositionStickyOnElement = true) {
+    this._borderCellCss = {
+      'top': `${_stickCellCss}-border-elem-top`,
+      'bottom': `${_stickCellCss}-border-elem-bottom`,
+      'left': `${_stickCellCss}-border-elem-left`,
```

Comment:
Would it be accurate to use "start" and "end" so that the terminology works with RTL?

Reply:
This is actually accurate - the rtl right/left swapping is already handled in the sticky column code below.